test/e2e/storage/testsuites/testdriver.go

```diff
@@ -112,6 +112,7 @@ type SnapshottableTestDriver interface {
 // Capability represents a feature that a volume plugin supports
 type Capability string
 
+//Constants related to capability
```

Comment:
nit: need extra space after `//`

Reply:
Thanks for the review. I added the space.